index.js

```diff
@@ -26,9 +26,16 @@ module.exports = function(options) {
         runOptions.push(file);
     });
 
-    runOptions.push('-f');
-    var format = options.format || 'pretty';
-    runOptions.push(format);
+    if (!options.format) {
+        options.format = 'pretty';
+    }
+
+    var formats = util.isArray(options.format) ? options.format : [options.format];
```

Comment:
Was just following the existing code right below this dealing with multiple tags. I can update it if needed.